services/brig/test/integration/API/Team.hs

```diff
@@ -138,11 +139,50 @@ testInvitationTooManyPending brig galley (TeamSizeLimit limit) = do
         const 403 === statusCode
         const (Just "too-many-team-invitations") === fmap Error.label . decodeBody
 
+-- | Admins can invite collaborators, but not owners.
+testInvitationRoles :: HasCallStack => Brig -> Galley -> Http ()
+testInvitationRoles brig galley = do
+    (owner, tid) <- createUserWithTeam brig galley
+
+    let registerInvite :: Invitation -> Email -> Http UserId
```

Comment:
There are no other top-level helpers in this module, no?  So it would just make this small function harder to find, and otherwise at best solve an aesthetic problem.  I would like ot keep this self-contained.